FloripaPublicTransportation/Base.lproj/LaunchScreen.storyboard

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="9532" systemVersion="15D21" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" launchScreen="YES" useTraitCollections="YES" initialViewController="01J-lp-oVM">
```

Comment:
Hello @arctouch-pedrocosta , yes, this was intentional. According to the Apple guidelines for launch files (https://developer.apple.com/library/ios/documentation/UserExperience/Conceptual/MobileHIG/LaunchImages.html), they tell us to "Design a launch image that is identical to the first screen of the app, except for UI elements that might change". That's what I did, my launch screen has all the basic content that the application will have once it is launched, except for the table view cells that will surely be replaced on runtime. Another interesting thing that Apple claims about launch images is that "the launch image isn’t an opportunity to provide an 'app entry experience,' such as a splash screen" - I say this is interesting because definitely, splash screens are what we see most often in the App Store apps. The link also has an example launch image that Apple uses for the Settings "app": as you can see, it is the bare minimum background of that app, something that will instantly get loaded before loading the UI specific contents. Thank you for your attention! :)